config/horizon.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+use Illuminate\Support\Str;
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Horizon Domain
+    |--------------------------------------------------------------------------
+    |
+    | This is the subdomain where Horizon will be accessible from. If this
+    | setting is null, Horizon will reside under the same domain as the
+    | application. Otherwise, this value will serve as the subdomain.
+    |
+    */
+
+    'domain' => null,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Horizon Path
+    |--------------------------------------------------------------------------
+    |
+    | This is the URI path where Horizon will be accessible from. Feel free
+    | to change this path to anything you like. Note that the URI will not
+    | affect the paths of its internal API that aren't exposed to users.
+    |
+    */
+
+    'path' => 'web/horizon',
```

Comment:
`/horizon` makes more sense - if the user is in the browser they are already making a web request.

Reply:
`/horizon` would mean that we couldn't use that as an API route - should be ok I think?

Reply:
I doubt we would need `/horizon` as an api route. Even if we did, we could handle that by working out the type of call made and returning the appropriate response

Reply:
10b7c84